compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -339,7 +339,7 @@ function parseIfElseStmt(Tokenizer tok, Position startPos) returns IfElseStmt|er
             IfElseStmt elseIfStmt = check parseIfElseStmt(tok, ifFalseStartPos);
             Position blockStartPos = elseIfStmt.ifTrue.startPos;
             StmtBlock? elseIfFalseBlock = elseIfStmt.ifFalse;
-            Position blockEndPos = (elseIfFalseBlock is StmtBlock)? (elseIfFalseBlock.endPos) : elseIfStmt.ifTrue.startPos;
+            Position blockEndPos = (elseIfFalseBlock is StmtBlock)? (elseIfFalseBlock.endPos) : elseIfStmt.ifTrue.endPos;
```

Comment:
Can we simplify this to `(elseIfFalseBlock ?: elseIfStmt.ifTrue).endPos`?

Reply:
fixed